sympy/integrals/manualintegrate.py

```diff
@@ -225,7 +226,7 @@ def test_subterm(u, u_diff):
         debug("substituted: {}, u: {}, u_var: {}".format(substituted, u, u_var))
         substituted = manual_subs(substituted, u, u_var).cancel()
 
-        if symbol in substituted.free_symbols:
+        if substituted.has(symbol):
             return False
```

Comment:
Why is this changed?

Should it not be `has_free` or `has_xfree` rather than `has`?